include/nbl/asset/metadata/CGLTFMetadata.h

```diff
@@ -31,6 +28,29 @@ class CGLTFMetadata final : public IAssetMetadata
             return static_cast<const asset::CGLTFPipelineMetadata*>(found);
         }
 
+        using INSTANCE_ID = uint32_t;
+
+        struct Scene
+        {
+            core::smart_refctd_ptr<asset::ICPUBuffer> instanceIDs;
+            uint32_t count;
+        };
+
+        struct Instance
+        {
+            const ICPUSkeleton* skeleton;
+            uint32_t skinTranslationTableOffset; // byteoffset into `vertexJointToSkeletonJoint`range
```

Comment:
remove `vertexJointToSkeletonJoint`, and `Instance::skinTranslationTableOffset` and replace it with an `SBufferBinding`

Reply:
corrected in following commit